package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I don't think this change is supposed to be here. @bbondy suggested that you might have an older version of npm. I get the same thing when I run init so I'm checking mine as well